Cargo.toml

```diff
@@ -18,3 +18,7 @@ clap = { version = "4.0.12", features = ["derive"] }
 fuzzy_finder = "0.3.0"
 shellexpand = "2.1.2"
```

Comment:
This is also a not sure dependency, since We are not handling things like `~` or `$HOME` for now